test/test_oneview_volume.py

```diff
@@ -429,7 +427,7 @@ def test_should_not_create_snapshot_when_oneview_exception(self, mock_ansible_mo
 
     @mock.patch.object(OneViewClient, 'from_json_file')
     @mock.patch('oneview_volume.AnsibleModule')
-    def test_should_not_delete_snapshot_when_oneview_exception(self, mock_ansible_module,
+    def test_should_fail_when_delete_snapshot_raises_exception(self, mock_ansible_module,
```

Comment:
The method that raises an exception is this case is 'create_snapshot', not delete_snapshot